models/model_choice.py

```diff
@@ -191,14 +212,18 @@ def net(net_params, num_channels, inference=False):
                         '4', nn.Conv2d(classifier[-1].in_channels, num_channels, kernel_size=(1, 1))
                 )
                 ###################
-                #conv1 = model.backbone._modules['conv1'].weight.detach().numpy()
-                #depth = np.random.uniform(low=-1, high=1, size=(64, 1, 7, 7))
-                #conv1 = np.append(conv1, depth, axis=1)
-                #conv1 = torch.from_numpy(conv1).float()
-                #model.backbone._modules['conv1'].weight = nn.Parameter(conv1, requires_grad=True)
+                # conv1 = model.backbone._modules['conv1'].weight.detach().numpy()
+                # depth = np.random.uniform(low=-1, high=1, size=(64, 1, 7, 7))
+                # conv1 = np.append(conv1, depth, axis=1)
+                # conv1 = torch.from_numpy(conv1).float()
+                # model.backbone._modules['conv1'].weight = nn.Parameter(conv1, requires_grad=True)
```

Comment:
Keeping this since our test13 model with deeplabv3 seems to have used this. Both solutions work but are not interchangeable. A model trained with this approach cannot be built afterwars with LayersEnsemble approach (e.g. for inference)